docs/src/APIs/Driver/index.md

```diff
@@ -3,12 +3,32 @@
 ```@meta
 CurrentModule = ClimateMachine
 ```
+## Solver types
+
+```@docs
+HEVISplitting
+MISSolverType
+MultirateSolverType
+AbstractSolverType
+DiscreteSplittingType
+ExplicitSolverType
+ConfigTypes
+IMEXSolverType
+```
+
+## Configurations
+
+```@docs
+DriverConfiguration
+SolverConfiguration
+```
+
+## CLI / arg parsing
```

Comment:
Should probably change the title of this, courant doesn’t fit well in this category